### ISSUE TITLE
Generate the country leader board by airflow

### ISSUE THREAD

**Issue by dadiorchen** [Greenstand/treetracker-airflow-dags#108]
Currently, the country leader board take too long time to generate, https://github.com/Greenstand/treetracker-query-api/issues/194 
So we consider to pre-generate the loader board by airflow. 
Basically, we have done this by javascript, now let's migrate to airflow with Python:

https://github.com/Greenstand/treetracker-query-api/blob/2c801a82d783acd5d92a6c4a6d8cb57430279e02/server/infra/database/CountryRepository.ts#L62-L212

Please output the data to: `webmap.config` as this form:

```
name = 'country-leader-board'
data = 
{
   global: [{country-obj, ...}],
   Africa: [...]
}
```

here, country-obj looks like: 

```
{
    //     planted: '1004',
    //     id: 6632436,
    //     name: 'Nigeria',
    //     centroid:
    //       '{"type":"Point","coordinates":[8.08943894770572,9.59411452232656]}',
    //   },
```

<img width="558" alt="image" src="https://user-images.githubusercontent.com/5744708/191476339-c635e970-15bb-4a87-a3f7-6236ae65fb6d.png">

**Comment by DerMoehre** [comment]
Hi, 
I would like to improve my python skills. 
I dont know anything about airflow, but I would like to contribute to this project. 

Could someone help me to get started? Like point in the right direction, or giving a helping hand :-)

**Comment by dadiorchen** [comment]
@DerMoehre welcome to greenstand, but another volunteer is working on this one, could you please take another one in the repo?

**Comment by DerMoehre** [comment]
@dadiorchen allright :) I will look through other issues.

**Comment by RubenSmn** [comment]
@dadiorchen is this issue resolved by #109?

**Comment by dadiorchen** [comment]
Yes, the country board is good now
